catalyst/dl/callbacks/phase.py

```diff
@@ -0,0 +1,112 @@
+from typing import List
+from collections import OrderedDict
+
+from catalyst.dl.core import Callback, CallbackOrder
+
+
+class Phase:
+    def __init__(self, name: str = None, steps: int = None):
+        self.steps = int(steps) if steps is not None else None
+        self.curr_step = 0
+        self.name = name
+
+
+class PhaseManager:
```

Comment:
docstring

Reply:
done. BTW initially I wanted to reply "no documentation is kind of catalyst culture=(", but when I merged, I've seen that things are changing slowly, that's great!